session.go

```diff
@@ -297,6 +297,7 @@ func ParseURL(url string) (*DialInfo, error) {
 			if err != nil {
 				return nil, errors.New("bad value for maxSocketReuseTime: " + v)
 			}
+			debugf("using maxSocketReuseTimeSecs: %d", maxSocketReuseTimeSecs)
```

Comment:
Yes I waned to keep it - its only visible while tests are running (or when we enable debug logging explicitly on app startup)